app/models/api-articles.model.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../../db/connection')
+
+exports.selectArticleById = (article_id) => {
+
+    return db.query('SELECT * FROM articles where article_id = $1;', [article_id])
+    .then((response) => {
+        if (response.rows.length === 0){
+            return Promise.reject({status: 404, message: 'Page Not Found'})
```

Comment:
We do not need to implement this logic because psql will throw an error if the id is not a number, it would be better to catch that error and delete this logic 🙂